docs/console.html

```diff
@@ -0,0 +1,804 @@
+x('html/body/p');
+</code></pre>
+            </figure>
+        </figure>
+    </section>
+
+    <section>
+        <h3 id="inspecting-dom-elements-and-javascript-heap-objects">Inspecting DOM elements and JavaScript heap objects</h3>
+        <p>
+            The <code><a href="commandline-api#inspectobject">inspect()</a></code> function takes a DOM element or JavaScript reference as a parameter.
+            If you provide a DOM element the DevTools will go to the Elements panel and display that element.
+            If a JavaScript reference is provided then it will go to the Profile panel.
+        </p>
+        <figure data-target="inspecting-dom-elements-example">
+            <figcaption>
+                When this code is executed in your console on this page, it will grab this figure and display it on the Elements panel. This is achieved by taking advantage of the <code<a href="commandline-api#_">$_</a></code> property to get the output of the last evaluated expression.
+            </figcaption>
+            <pre class="prettyprint"><code class="language-javascript">
+$('[data-target="inspecting-dom-elements-example"]')
+inspect($_)
+</code></pre>
+        </figure>
+    </section>
+
+    <section>
+        <h3 id="accessing-recently-selected-elements-and-objects">Accessing recently selected elements and objects</h3>
+        <p>
+            The console has access to the last five elements or objects that were selected.
+            As you select an element in the Elements panel or an object in the Profiles panel, that is pushed onto the history stack.
+            The history stack is accessed by using <code><a href="commandline-api#0-4">$<var>x</var></a></code> where <var>x</var> is the number of the item you wish to access.
+            Remember computers begin counting from 0; this means the first item is <code>$0</code> and the last item is <code>$4</code>.
+        </p>
+    </section>
+
+    <section>
+        <h3 id="monitoring-events">Monitoring events</h3>
+        <p>
+            The <code><a href="commandline-api#monitoreventsobject-events">monitorEvents()</a></code> method instructs the DevTools to log information on the specified targets.
+            The first parameter is the object to monitor.
+            To specify the events to listen to you may pass either a string or an array of strings as the second parameter.
+        </p>
+        <figure>
+            <figcaption>Listen to click events on the body of the page.</figcaption>
+            <pre class="prettify"><code class="language-javascript">
+monitorEvents(document.body, "click");
+</code></pre>
+        </figure>
+        <p>
+            If a supported <i>event type</i> that the DevTools maps to a set of standard event names is provided then it will listen to all of the events for that type.
+            A full list of <i>event types</i> can be found in the <a href="commandline-api#monitoreventsobject-events">Command Line API</a>.
```

Comment:
All the links in the preview 404... The link this is for [does exist](https://developer.chrome.com/devtools/docs/commandline-api#monitoreventsobject-events) in our current docs... Which also needs a refresh. Look at those images! So old.